src/formattedcode/templates/html-app/template.html

```diff
@@ -37,33 +37,33 @@
     <link href="{{ assets_dir }}/simple-sidebar.css" rel="stylesheet"/>
 </head>
 <body>
-
-<div class="container-fluid">
-    <nav class="navbar navbar-default">
-      <div class="container-fluid">
-        <div class="navbar-header">
-            <span class="navbar-brand icon-primary"><a href="https://github.com/nexB/scancode-toolkit/" target="_blank">ScanCode</a> scan results for: {{ scanned_path }}</span>
-        </div>
-          <ul class="nav navbar-nav navbar-right">
-            <li><a href="{{ assets_dir }}/help.html" target="_blank">Help</a></li>
-            <li><a href="http://www.nexb.com" target="_blank">Made by nexB</a></li>
-         </ul>
-      </div>
-    </nav>
-    <!--
+  <nav class="navbar navbar-inverse navbar-static-top">
```

Comment:
I never saw this <nav> tag before? Is this standard on all browsers starting IE8 and up? afaik this is only HTML5. If yes, we cannot use this.